.circleci/config.yml

```diff
@@ -938,25 +963,43 @@ workflows:
   launch-all-app-builds-workflow:
     # using do-builds param to select which of the two workflows using the build-on-tag-filters
     # should run; should be this one that just launches the builds or the one actually doing the builds
-    unless: << pipeline.parameters.do-builds >>
+    when:
+      and:
+        - not: << pipeline.parameters.do-builds >>
+        - equal: [ UNKNOWN, << pipeline.parameters.api_call >> ]
     jobs:
-      - conditionally-launch-build-and-store-all-job: &build-and-store-filters
-          filters:
-            branches:
-              only:
-                - /^rc.*/
-                - /^hotfix.*/
+      - api-unit-test:
+          <<: *filter-release-branch
+      - ui-unit-test:
+          <<: *filter-release-branch
+          name: << matrix.study_key >>-ui-unit-test
+          matrix:
+            alias: app-ui-unit-test
+            parameters:
+              <<: *studies
+      - app-build:
+          <<: *filter-release-branch
+          name: << matrix.study_key >>-build
+          matrix:
+            alias: app-build
+            parameters:
+              <<: *studies
+          check_archive: true
+          store_build: true
+          requires:
+            - << matrix.study_key >>-ui-unit-test
+            - api-unit-test
```

Comment:
Note, it would be very easy to add a new job to run Playwright tests as prerequisite for build if we wanted to. Question, do we want to add that?